src/Arcus.Scripting.IntegrationAccount/Scripts/Set-AzIntegrationAccountAssemblies.ps1

```diff
@@ -0,0 +1,73 @@
+Param(
+    [Parameter(Mandatory = $true)][string] $ResourceGroupName = $(throw "Resource group name is required"),
+    [Parameter(Mandatory = $true)][string] $Name = $(throw "Name of the Integration Account is required"),
+    [parameter(Mandatory = $false)][string] $AssemblyFilePath = $(if ($AssembliesFolder -eq '') { throw "Either the file path of a specific assembly or the file path of a folder containing multiple assemblies is required, e.g.: -AssemblyFilePath 'C:\Assemblies\assembly.dll' or -AssembliesFolder 'C:\Assemblies'" }),
+    [parameter(Mandatory = $false)][string] $AssembliesFolder = $(if ($AssemblyFilePath -eq '') { throw "Either the file path of a specific assembly or the file path of a folder containing multiple assemblies is required, e.g.: -AssemblyFilePath 'C:\Assemblies\assembly.dll' or -AssembliesFolder 'C:\Assemblies'" }),
+    [Parameter(Mandatory = $false)][string] $ArtifactsPrefix = ''
+)
+
+if ($AssemblyFilePath -ne '' -and $AssembliesFolder -ne '') {
+    throw "Either the file path of a specific assembly or the file path of a folder containing multiple assemblies is required, e.g.: -AssemblyFilePath 'C:\Assemblies\assembly.dll' or -AssembliesFolder 'C:\Assemblies'"
+}
+
+function UploadAssembly {
+    param
+    (
+        [System.IO.FileInfo][parameter(Mandatory = $true)]$Assembly
+    )
+
+    $assemblyName = $Assembly.BaseName
+    if ($ArtifactsPrefix -ne '') {
+        $assemblyName = $ArtifactsPrefix + $assemblyName
+    }
+    Write-Host "Uploading assembly '$assemblyName' into the Integration Account '$Name'."
```

Comment:
I think that the other logging statements also don't use points at the end of their sentences. Maybe we should remove them here too, then.

Reply:
I copied this from the other Set-AzIntegrationAccount scripts, they contain the points at the end as well, for example:
`Write-Verbose "Checking if the map '$mapName' already exists in the Integration Account '$Name'."` and
`Write-Host "Uploading schema '$schemaName' into the Integration Account '$Name'."`

No problem to change it, but then I will also need to change the already existing scripts to upload schemas and maps.

Reply:
Shouldn't have handed me that 🥇 just yet, probably 😉

Reply:
Keep in mind if this gets updated in the script, to also update the docs

Reply:
Removed the point in the logging statements, changed the docs as well, see https://github.com/arcus-azure/arcus.scripting/pull/257/commits/7863ca153a912cc4ff015e5bf64e15059416bbe0